scripts/services/map_manager.gd

```diff
@@ -3,11 +3,17 @@ extends Node
 const MAX_MAP_SIZE = 40
 const LIST_FILE_PATH = "user://known_maps_list.json"
 const MAP_PATH = "user://"
+const BUNDLED_MAP_PATH = "res://assets/maps/"
 const MAP_EXTENSION = ".tofmap.json"
+const MAIN_MENU_MAP_NAME = "main_menu_bg"
 
 var filesystem = preload("res://scripts/services/filesystem.gd").new()
 
 var maps = {}
+var bundled_maps = {
+    "crossroad": {"name": "crossroad", "online_id" : null}, 
+    "crossroad2": {"name": "crossroad2", "online_id" : null}
+}
```

Comment:
Maybe add `"bundled" : true` attribute to these maps? Could be used later in the map selection menu to somehow indicate, that map is bundled as opposed to custom or downloaded. Alternatively, a `is_bundled(map_name)` method could be used for that, and it could help making logic in some of the other methods more readable.

Reply:
I was thinking about this. Yes, it's a good idea